packages/integration-karma/test/light-dom/synthetic-shadow/index.spec.js

```diff
@@ -0,0 +1,64 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
+import Container from 'x/container';
+
+// This test only matters for synthetic shadow
+if (!process.env.NATIVE_SHADOW) {
+    describe('Light DOM and synthetic shadow', () => {
+        beforeEach(() => {
+            setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', true);
+        });
+        afterEach(() => {
+            setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', false);
+        });
+        it('shadowToken and domManual are not set for shadow components', () => {
+            // shadow grandparent, light child, shadow grandchild
+            const elm = createElement('x-container', { is: Container });
+            document.body.appendChild(elm);
+
+            // shadow grandparent
+            expect(elm.shadowRoot.querySelector('h1').$shadowToken$).toEqual(
+                'x-container_container'
+            );
```

Comment:
I noticed that no other karma test checks for `$shadowToken$` or `$domManual$`. I don't know if there's a reason for that, but this seemed to be the most straightforward way to check if the attributes or MutationObservers were being added.

Reply:
The `$shadowToken$` and `$domManual$` are implementation details for synthetic shadow DOM scoping. The `$shadowToken$` property is reflected to an attribute (and soon a class). 

Instead of checking those properties, I would recommend testing the element `innerHTML` value. It shouldn't contain any scoping attribute (or class).

Reply:
Done.